rhine/src/FRP/Rhine/Clock/Never.hs

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE TypeFamilies #-}
+
+-- | A clock that never ticks.
+module FRP.Rhine.Clock.Never where
```

Comment:
```suggestion
module FRP.Rhine.Clock.Realtime.Never where
```

To fit better in the existing module hierarchy, I believe this clock should be in the `Realtime` namespace.

Reply:
Yup, makes sense since we're using `UTCTime`.